### ISSUE TITLE
Different names for secret referenced in "Making cert-manager work with HTTPProxy"

### ISSUE THREAD

**Issue by vukg** [projectcontour/contour#2281]
**What steps did you take and what happened:**
In the documentation the guide for cert-manager https://projectcontour.io/guides/cert-manager/ gives solution how could we proceed with "Making cert-manager work with HTTPProxy"

The workaround builds on using Ingress object to create Secret which is than consumed by HTTPProxy by referencing the same secret. 

However, example specifies different names for Secret in Ingress and HTTPProxy objects.

```yaml class:"lineNo"
apiVersion: projectcontour.io/v1
kind: HTTPProxy
metadata:
  name: httpbinproxy
spec:
  virtualhost:
    fqdn: httpbinproxy.davecheney.com
    tls:
      secretName: httpbin
  routes:
  - services:
    - name: httpbin
      port: 8080
```
``` yaml
apiVersion: networking.k8s.io/v1beta1
kind: Ingress
metadata:
  annotations:
    cert-manager.io/cluster-issuer: letsencrypt-prod
    ingress.kubernetes.io/force-ssl-redirect: "true"
    kubernetes.io/tls-acme: "true"
  name: httpbinproxy
  namespace: default
spec:
  rules:
  - host: httpbinproxy.davecheney.com
    http:
      paths:
      - backend:
          serviceName: httpbin
          servicePort: 8080
  tls:
  - hosts:
    - httpbinproxy.davecheney.com
    secretName: httpbinproxy
```

Ingress creates secretName: **httpbin**, while HTTPProxy defines secretName: **httpbinproxy**.

**What did you expect to happen:**

Both Ingress and HTTPPRoxy should define Secret with same name e.g. secretName: **httpbin**

**Comment by stevesloka** [comment]
Hey @vukg you are correct, the docs at the last step reference a different secret. Would you like to send a PR to update? If not I can help take that and get it fixed up. 

Also, another way to accomplish this would be to use a CertManager CRD type `Certificate` which would do the same request flow and eliminates that extra Ingress object sitting around.

**Comment by vukg** [comment]
Hey @stevesloka tnx for feedback! I could create PR. Just need to find where the source is, because it is moved out of main tree. If you can point me to right place it would be great.

**Comment by stevesloka** [comment]
You'd want to update the following file: https://github.com/projectcontour/contour/blob/master/site/_guides/cert-manager.md

**Comment by vukg** [comment]
Hi @stevesloka I saw 1.3.0 out but without this PR. Should I redo something to make it easier?

**Comment by jpeach** [comment]
Fixed in  #2284.
